simulators/ethereum/eest/execute/Dockerfile

```
@@ -0,0 +1,26 @@
+# Builds and runs the EEST (execution-spec-tests) execute hive simulator
+FROM ghcr.io/astral-sh/uv:python3.10-bookworm-slim
+
+## Default fork/marker/git-ref
+ARG fork=Prague
+ENV FORK=${fork}
+ARG marker=
+ENV MARKER=${marker}
```

We're kinda leaking pytest-speak into the hive world (what's a marker?), which I'd prefer to avoid, but I don't think it's possible, so...

In general this could be a marker expression, this is a big ugly, but would better convey this possibility:
```suggestion
ARG marker_expr=
ENV MARKER_EXPR=${marker_expr}
```

Would be awesome to provide help from the command-line on these possible args, don't know how yet tho.

Btw, from the pytest help:
```
  -m MARKEXPR           Only run tests matching given mark expression. For
                        example: -m 'mark1 and not mark2'.
```